src/test/compile-fail/issue-38412.rs

```diff
@@ -11,6 +11,7 @@
 fn main() {
     let Box(a) = loop { };
     //~^ ERROR expected tuple struct/variant, found struct `Box`
+    //~| ERROR expected tuple struct/variant, found struct `Box`
```

Comment:
This is a compiletest bug. If one error has two labels, compiletest will require two "expected errors".